### DISCUSSION TITLE
GDMP for web

### DISCUSSION THREAD

**Discussion by Ehsanwww** [j20001970/GDMP#27] [General]
Hello @j20001970 , thanks for great asset, please provide tutorial for web GDMP. thanks again

**Comment by j20001970** [comment]
The web build for GDMP is still under development, you can see the progress on #23. Although it has been stalled for quite a while, as I've been busy on irl stuff.

**Reply by Ehsanwww** [reply]
@j20001970 Thanks,
there is no web build for both godot 3 and godot 4? right? if yes, do you think when it will be ready ?

**Reply by Ehsanwww** [reply]
please tell me if I can help you  and contribute about web export  @j20001970

**Reply by j20001970** [reply]
> @j20001970 Thanks, there is no web build for both godot 3 and godot 4? right? if yes, do you think when it will be ready ?

The `web` branch is based on Godot 4. We will see the possibility of backporting to Godot 3 when it is completed.
Unfortunately, I can't give a ETA about web version for GDMP, but if you would like to contribute, you can open PRs that merge to `web` branch, I'll review them as soon as possible.

**Reply by Ehsanwww** [reply]
@j20001970 actually I mean web export for godot 4, I compiled it and I got error

**Comment by j20001970** [comment]
Hi, it's been a while but I would like to inform you that GDMP for Web is now available since v0.5 (only on Godot 4.x at the moment but still), there is [live demo](https://github.com/j20001970/GDMP-demo) for Web available if you would like to try it out. I think this discussion can be closed now, please open separate issue if you have problem using GDMP on Web platform.
